src/components/vipps-login-button/index.tsx

```diff
@@ -0,0 +1,54 @@
+import {LoginTexts, useTranslation} from '@atb/translations';
+import React from 'react';
+import ThemeText from '@atb/components/text';
+import ThemeIcon from '@atb/components/theme-icon';
+import {TouchableOpacity, View} from 'react-native';
+import {StyleSheet, Theme} from '@atb/theme';
+import vippsLogo from '@atb/assets/svg/color/icons/ticketing/VippsLogo';
+
+export const VippsLoginButton = ({
+  onPress,
+  disabled,
+}: {
+  onPress: () => {};
+  disabled: boolean;
+}) => {
+  const {t} = useTranslation();
+  const style = useStyle();
+  const getVippsLabel = () => (
```

Comment:
Why is this defined as a function instead of just used directly?

Reply:
yes it could be..it was needed earlier but not anymore..can fix it 👍